src/GTP.cpp

```diff
@@ -67,6 +67,8 @@ FILE* cfg_logfile_handle;
 bool cfg_quiet;
 std::string cfg_options_str;
 
+std::unique_ptr<UCTSearch> search = nullptr;
```

Comment:
Use uniform initialization: std::unique_ptr<UCTSearch> search{nullptr};

This can also be a static variable inside GTP::execute, which reflects ownership better, and which means it will only get its constructor called when cfg_* is already set up.

Reply:
fixed